app/components/public/speaker-item.js

```diff
@@ -1,12 +1,7 @@
 import Ember from 'ember';
 
-const { Component, computed } = Ember;
+const { Component } = Ember;
 
 export default Component.extend({
-  classNameBindings: ['style'],
-
-  style: computed('isFeatured', function() {
-    const allClasses = this.get('isFeatured') ? 'column eight wide' : 'column four wide';
-    return allClasses;
-  })
+  classNames: ['eight wide speaker column']
```

Comment:
We want two column only for 480px to 768px. The rest should use the default behavior.